CTA DC1 Vela phasogram

msjacob opened the pull request:
In the repository datasets/pulsar, three new files: Vela (J0835-4510) phasecurve and two DC1 fits file of simulated Vela data, with an added phase column in it. We can see the Vela lightcurve appear as we make a histogram on the phases.

cdeil commented:
Thanks!

I'm merging this now.

Do you mean you added a PHASE column to the event lists?
If yes, would it be possible to add the script that did this here?
It will help with testing / documentation, to know how the PHASE was filled.

msjacob commented:
Yes I added a phase column to the event lists.
At the time I did it with Ctools but then I realized the phasing was actually very simple for the DC1 and it was short to write in python. I added a python file about that if it can help.
Thank you!
